guides/common/modules/proc_configuring-podman-to-trust-the-ca.adoc

```diff
@@ -0,0 +1,33 @@
+[id="Configuring_podman_to_trust_the_CA_{context}"]
+= Configuring Podman and Docker to trust the Certificate Authority
+
+Podman uses two paths to locate the CA file, namely, `/etc/containers/certs.d/` and `/etc/docker/certs.d/`.
+
+* You might first need to create the location using:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# mkdir -p /etc/containers/certs.d/hostname.example.com
+----
+* Copy the root CA file to one of these locations, with the exact path determined by the server hostname, either `{project-context}-server` or `{smart-proxy-context}-server`, and naming the file `ca.crt`:
```

Comment:
Users are free to choose hostnames, so I think `either` is too strong. Something like `for example` or `such as` might be better.